src/hacbs/types/coreBuildService.ts

```diff
@@ -51,13 +51,18 @@ export type Env = {
   value: string;
 };
 
-export type ReleaseLinkKind = K8sResourceCommon & {
-  spec: ReleaseLinkSpec;
+export type ReleasePlanKind = K8sResourceCommon & {
+  spec: ReleasePlanSpec;
 };
 
-export type ReleaseLinkSpec = {
+export type ReleasePlanSpec = {
   displayName: string;
   application: string;
-  target: string;
+  target: ReleaseTarget;
```

Comment:
should this be `ReleaseTarget.namespace` for now?

Reply:
I don't think so - I think this defines a new type, and this line indicates that the `target` of the `ReleasePlanSpec` should be another typed object. That type is defined just below this one, and it contains a mandatory `namespace` and an optional `workspace`.